packages/components/src/components/Log/Log.test.js

```diff
@@ -67,6 +67,58 @@ describe('Log', () => {
     await waitFor(() => getByText('Line 1'));
   });
 
+  it('auto-scrolls down, for a running step, when new logs are added and the Log component bottom is below the viewport', async () => {
+    jest
+      .spyOn(Element.prototype, 'getBoundingClientRect')
+      .mockReturnValue({ bottom: 2500 })
+      .mockReturnValueOnce({ bottom: 0 });
+    const spiedFn = jest.spyOn(Element.prototype, 'scrollTop', 'set'); // the scrollTop value is changed in scrollToBottomLog
+    render(
+      <Log
+        stepStatus={{ running: true }}
+        fetchLogs={() => 'Line 1'}
+        enableLogAutoScroll
+      />
+    );
+    await waitFor(() => expect(spiedFn).toBeCalled());
```

Comment:
nit: `toHaveBeenCalled()` for consistency with existing tests

Reply:
changed as requested in the amended commit